src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -118,6 +130,18 @@ def __init__(
                     unrealized_coef=1.0,
                     discount_factor=self.discount_factor,
                 )
+            elif self.aggressiveness is None:
```

Comment:
I don't think you can ever hit this case when using the CLI. You should probably make a `custom` option or something similar.

Reply:
Since the default value of aggressiveness is set to None, I believe it should hit whenever no argument is passed into argparse (since the default return for argparse is None if a commandline argument is not included)

Reply:
The default return value is `None` only if the `default` parameter is not set. However, it *is* set to `Aggressiveness.moderate`.

I'm suggesting a `custom` option so the default can be kept the way it was before.

Reply:
I can definitely do that. I was just worried that that would require additional text to be passed in the command and might be confusing to a commandline user that isn't you or I.

Reply:
If you are already passing in all these coefficients, I don't think that adding `--agressiveness custom` would be a lot more.

Adding default values for the coefficients in `argparse` might be a simpler solution, though. The validation logic required would be a lot less complicated.